packages/eslint-rules/index.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * ESLint rules.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
```

Comment:
Can we structure this a bit closer to `@wordpress/eslint-plugin` ? This is currently named as `eslint-rules`, but I think it makes sense to model the package in the same way.

I think we can leave refactoring our setup to put more/all/most of our ESLint config into the plugin for another issue, but I don't see why we can't structure it in the same way to start?

E.g. 
* this file – https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/index.js
* rules index – https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/rules/index.js
* individual rule – https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/rules/i18n-text-domain.js

What do you think?

Reply:
Works for me. I'm making the change 👍🏻